### DISCUSSION TITLE
Welcome to Mol* Discussions!

### DISCUSSION THREAD

**Discussion by dsehnal** [molstar/molstar#109] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by agdturner** [comment]
Hi, I am working with some biomolecular researchers from Jan to Aug 2025 (possibly longer). The researchers want a tool to help them compare Ribosome data. There are a few requirements and Mol* is a great starting point. I'm looking forward to learning what Mol* can do already, what the development roadmap is, and how to contribute and develop something sustainably. :)

**Reply by dsehnal** [reply]
Sounds good. What functionality are you after?

For roadmap map, no formal plans overall. Informal: WebGPU support, extending https://molstar.org/mol-view-spec/, continued bugfixes, supporting community contributions.

**Reply by agdturner** [reply]
A key thing is that the researchers want to be able to load different ribosome data sets into different "viewers", align the molecular structures and then link the viewers so that these can be explored interactively (where for example zooming or twisting one molecule does the same to the other).
